test/cli_stop_test.go

```diff
@@ -126,3 +126,21 @@ func (suite *PouchStopSuite) TestStopMultiContainers(c *check.C) {
 	c.Assert(string(result[0].State.Status), check.Equals, "stopped")
 
 }
+
+// TestStopPidValue ensure stopped container's pid is 0
+func (suite *PouchStopSuite) TestStopPidValue(c *check.C) {
```

Comment:
please add one case for the process exits by itself.

Reply:
make sense